.github/workflows/pre-release.yml

```diff
@@ -62,6 +62,13 @@ jobs:
           echo "" >> "${{ env.RELEASE_NOTES_FILE }}"
           echo "**GitHub Actions Run:** $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> "${{ env.RELEASE_NOTES_FILE }}"
 
+      - name: Temporarily disable "include administrators" branch protection
+        uses: benjefferies/branch-protection-bot@1.0.7
```

Comment:
I would prefer to pin it to a specific commit hash, this way we are **exactly** sure which version we are running.

Reply:
Done ✔️